libs/pilight/lua_c/lua.h

```diff
@@ -11,9 +11,9 @@
 
 #include <setjmp.h>
 
-#include <luajit-2.0/lua.h>
-#include <luajit-2.0/lualib.h>
-#include <luajit-2.0/lauxlib.h>
+#include <luajit-2.1/lua.h>
```

Comment:
Can you leave these unchanged because else my own build process fails.

Reply:
Sure, I'll revert them.